src/main/webapp/i18n/de/textExercise.json

```diff
@@ -29,7 +29,17 @@
             "checkPlagiarism": "Auf Plagiate prüfen",
             "exampleSubmissionsRequireExercise": "Um eine Beispielabgabe zu erstellen, musst du zunächst die Erstellung der Textaufgabe abschließen. Du wirst automatisch weitergeleitet.",
             "saveSuccessful": "Dein Antwort wurde erfolgreich gespeichert!",
-            "error": "Ein Fehler ist aufgetreten. Bitte versuche es später noch einmal."
+            "error": "Ein Fehler ist aufgetreten. Bitte versuche es später noch einmal.",
+            "clusterStatistics": {
+                "title": "Cluster Statistik",
+                "clusterId": "Cluster ID",
+                "reusedFeedbackRatio": "Quote wiederverwendeter Feedbacks im Cluster",
+                "reusedFeedbackRatioTooltip": "Anzahl der wiederverwendeten automatischen Feedbacks / Clustergröße",
```

Comment:
I am pretty sure, that there is no plural of "Feedback" (neither in German nor in English)
I would suggest: "Verhältnis von wiederverwendetem Feedback im Cluster und Clustergröße", "Anzahl wiederverwendetes Feedback / Clustergröße"